sdk/confidentialledger/Azure.ResourceManager.ConfidentialLedger/api/Azure.ResourceManager.ConfidentialLedger.netstandard2.0.cs

```diff
@@ -171,4 +218,67 @@ public partial class ConfidentialLedgerProperties
+    public readonly partial struct LanguageRuntime : System.IEquatable<Azure.ResourceManager.ConfidentialLedger.Models.LanguageRuntime>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public LanguageRuntime(string value) { throw null; }
+        public static Azure.ResourceManager.ConfidentialLedger.Models.LanguageRuntime CPP { get { throw null; } }
+        public static Azure.ResourceManager.ConfidentialLedger.Models.LanguageRuntime JS { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.ConfidentialLedger.Models.LanguageRuntime other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.ConfidentialLedger.Models.LanguageRuntime left, Azure.ResourceManager.ConfidentialLedger.Models.LanguageRuntime right) { throw null; }
+        public static implicit operator Azure.ResourceManager.ConfidentialLedger.Models.LanguageRuntime (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.ConfidentialLedger.Models.LanguageRuntime left, Azure.ResourceManager.ConfidentialLedger.Models.LanguageRuntime right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class ManagedCCFProperties
+    {
+        public ManagedCCFProperties() { }
+        public string AppName { get { throw null; } }
+        public System.Uri AppUri { get { throw null; } }
+        public Azure.ResourceManager.ConfidentialLedger.Models.DeploymentType DeploymentType { get { throw null; } set { } }
+        public System.Uri IdentityServiceUri { get { throw null; } }
+        public System.Collections.Generic.IList<Azure.ResourceManager.ConfidentialLedger.Models.MemberIdentityCertificate> MemberIdentityCertificates { get { throw null; } }
+        public int? NodeCount { get { throw null; } set { } }
+        public Azure.ResourceManager.ConfidentialLedger.Models.ConfidentialLedgerProvisioningState? ProvisioningState { get { throw null; } }
+    }
+    public partial class MemberIdentityCertificate
+    {
+        public MemberIdentityCertificate() { }
+        public string Certificate { get { throw null; } set { } }
+        public string Encryptionkey { get { throw null; } set { } }
+        public System.BinaryData Tags { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct RunningState : System.IEquatable<Azure.ResourceManager.ConfidentialLedger.Models.RunningState>
```

Comment:
The type name is too generic, use `prepend-rp-prefix` to specify it.

Reply:
Done, Thanks!